taxi/views.py

```diff
@@ -87,3 +104,30 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverForm
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
```

Comment:
As per requirements you should keep only one view with Driver Update - that will be one with only driver license update

Reply:
In this task the exercise is about correctly implementing overriding of form on UpdateView. And what is the point of having a full update and also update of one field?

Reply:
OK. I have deleted unnecessary updating form, view and link. Now everything (even updating license) works like it should.